sdks/python/apache_beam/runners/worker/opcounters_test.py

```diff
@@ -41,6 +46,59 @@ def __init__(self):
     pass
 
 
+class TransformIoCounterTest(unittest.TestCase):
+
+  def setUp(self):
+    try:
+      # pylint: disable=global-variable-not-assigned
+      global statesampler
+      from apache_beam.runners.worker import statesampler
+    except ImportError:
+      raise SkipTest('State sampler not compiled.')
+    super(TransformIoCounterTest, self).setUp()
+
+  def test_basic_counters(self):
+    counter_factory = CounterFactory()
+    sampler = statesampler.StateSampler('stage1', counter_factory)
+    sampler.start()
+
+    with sampler.scoped_state('step1', 'stateA'):
+      counter = opcounters.SideInputReadCounter(counter_factory, sampler,
+                                                declaring_step='step1',
+                                                input_index=1)
+    with sampler.scoped_state('step2', 'stateB'):
+      with counter:
+        counter.add_bytes_read(10)
+        sleep(0.3)
```

Comment:
<!--new_thread; commit:c1c26b3fd7c3a3eba913bfa6b714db6a687754a6; resolved:0-->
Can you comment that this is just an arbitrary short interval (or otherwise, how this value was chosen)?